ossfuzz/compress_frame_fuzzer.c

```diff
@@ -10,12 +10,14 @@
 #include <string.h>
 
 #include "fuzz_helpers.h"
+#include "fuzz_data_producer.h"
 #include "lz4.h"
 #include "lz4frame.h"
 #include "lz4_helpers.h"
 
 int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
 {
+    FUZZ_dataProducer_t *producer = FUZZ_dataProducer_create(data, size);
```

Comment:
This file should not have been changed. Will do this fuzzer later. I'm going to undo the changes in the next commit.